fern/docs/pages/feature-management/feature-types.mdx

```diff
@@ -0,0 +1,39 @@
+---
+title: Feature Types
+slug: feature-management/feature-types
+description: The 3 different types of features in Schematic -- Boolean, Trait-based, and Event-based.
+---
+
+There are 3 different types of features in Schematic
+- **Boolean** - A simple on/off feature.
+- **Event-based** - A feature that is metered against user events reported to Schematic (e.g. api calls, reports generated).
+- **Trait-based** - A feature that is based off of information reported to Schematic. This can be usage (e.g. seats) or metadata (e.g. company size).
+
+## Boolean Features
+
+Boolean features are the simplest type of feature. They are often used to represent features that are either available or not available to a user. They are the most commonly used type of feature and are the backbone of most plans. 
```

Comment:
I would maybe say "to a user or company" here, as flags are often checked in the context of a company only (backend/non-web context, like how we use it to enable certain import behaviors for Stripe)